lib/src/compiler.dart

```diff
@@ -36,6 +36,7 @@ import 'info.dart'
     show AnalyzerMessage, CheckerResults, LibraryInfo, LibraryUnit;
 import 'options.dart';
 import 'report.dart';
+import 'package:dev_compiler/src/server/server.dart';
```

Comment:
I'd also be tempted to "show DevServer" to make it clear that's the only part used